crates/llm-base/src/model/mod.rs

```diff
@@ -205,8 +205,18 @@ pub trait Model: Send + Sync {
     /// [InferenceSession::infer]). This value is configured through
     /// [ModelParameters::inference_parameters].
     fn inference_parameters(&self) -> &InferenceParameters;
+
+    /// Clone this model into a boxed trait object.
+    fn clone_box(&self) -> Box<dyn Model>;
```

Comment:
Incredibly minor: I prefer `clone_boxed` just to indicate that the clone is being boxed, not that the box itself is being cloned